test/pensions/config/employmentHistory.unit.spec.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import moment from 'moment';
+import { getFormDOM } from '../../util/schemaform-utils';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/pensions/config/form';
+
+describe('Pensions employment history', () => {
+  const { depends, schema, uiSchema } = formConfig.chapters.workHistory.pages.employmentHistory;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(2);
+  });
+
+  it('should expand to show all questions', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    formDOM.setYesNo('#root_view\\:workedBeforeDisabledYes', true);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(18);
+  });
+
+  it('should not submit empty form', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = getFormDOM(form);
+    formDOM.setYesNo('#root_view\\:workedBeforeDisabledYes', true);
```

Comment:
This doesn't seem notably better than `formDOM.fillData('#root_view\\:workedBeforeDisabledYes', 'Y');`